mmv1/products/compute/api.yaml

```diff
@@ -4214,6 +4214,16 @@ objects:
           The internal fully qualified service name for this Forwarding Rule.
           This field is only used for INTERNAL load balancing.
         output: true
+      - !ruby/object:Api::Type::Enum
+        name: 'ipVersion'
+        description: |
+          The IP address version that will be used by this forwarding rule.
+          Valid options are IPV4 and IPV6.
+
+          If not set, the IPv4 address will be used by default.
+        values:
+          - :IPV4
+          - :IPV6
```

Comment:
```suggestion
```

I was conflicted about whether to keep this new yaml here or to delete it. On one hand it could contribute to users being confused about what code controls the forwarding rule resource (in magic-modules/mmv1/ versus magic-modules/tpgtools). But, on the other hand, the code in tpgtools is deprecated so this new yaml here _could_ become what controls the resource in future.

I think it's better to remove the yaml as we can try to minimise confusion now, but the other thing is just hypothetical at the moment. Sorry for asking you to delete your work!